recipes/fps-auth-fief/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "fps-auth-fief" %}
+{% set version = "0.0.37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fps_auth_fief-{{ version }}.tar.gz
+  sha256: 14179c8383c36480151ec5cf0c83826c981d75566f0ffd35c179e1d4bc4dd0e0
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - fps >=0.0.8,<1
+    - fief-client-fastapi
+
+test:
+  imports:
+    - fps_auth_fief
```

Comment:
do you want to add `commands: - pip check`?

Reply:
I always forget, it's done now!